ivy.xml

```diff
@@ -90,8 +90,8 @@
                 rev="${apache-rat-tasks.version}" conf="releaseaudit->default">
         <exclude org="commons-collections" module="commons-collections"/>
     </dependency>
-    <dependency org="commons-lang" name="commons-lang" 
```

Comment:
This seems unrelated

Reply:
It appears the commons-lang library is not included for all the targets in the ivy settings [https://github.com/apache/zookeeper/blob/master/ivy.xml#L93]. So I have to update the conf with `default` to make the code compile. Also the apache commons-lang is not used (`org.apache.commons.lang`) is not used anywhere in the source. So this might not have been a issue before.